packages/login/src/vaadin-login-overlay.js

```diff
@@ -50,7 +50,7 @@ import { LoginMixin } from './vaadin-login-mixin.js';
  * @mixes LoginMixin
  * @mixes OverlayClassMixin
  */
-class LoginOverlay extends LoginMixin(OverlayClassMixin(ElementMixin(ThemableMixin(PolymerElement)))) {
+class LoginOverlay extends LoginOverlayMixin(OverlayClassMixin(ElementMixin(ThemableMixin(PolymerElement)))) {
```

Comment:
`OverlayClassMixin` is already applied by `LoginOverlayMixin`. Same goes for the TS definitions.

Reply:
Thanks, fixed.